### DISCUSSION TITLE
Integration to Freqtrade

### DISCUSSION THREAD

**Discussion by jothi-devhub** [hootnot/saxo_openapi#22] [Ideas]
I am trying integrate Saxo Open API to FreqTrade.As part of this I need to add the api to CCXT library need code verification. Please review here [forked_ccxt](https://github.com/jothi-devhub/ccxt/tree/master/python/ccxt) and provide your suggestions.

**Comment by hootnot** [comment]
Hi,

sorry but I don't have the time.  But while browsing a few files I saw both '_openapi_' and '_openai_'  in abstract/saxo.py.

**Reply by jothi-devhub** [reply]
Thank you
